### PR TITLE
Add forget password button

### LINKED ISSUE #23: Add forget password button to login activity
button should call firebase auth forget password function.

### PR DESCRIPTION
**Pull request by master-faz**
- Added forget password button to login

- sends email to reset password

Closes issue #23

### CODE REVIEW

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -23,7 +23,6 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
-            android:layout_below="@+id/studyHopperLogo"
```
Comment (mkaburis) [inline app/src/main/res/layout/activity_login.xml:26]: Is the logo gone from this activity's layout?

**Review by mkaburis — COMMENTED** [review]
Looks good! I just left one comment about the logo in activity_login.xml.

File: app/src/main/res/layout/activity_login.xml (hunk 1 above)
Comment (master-faz) [reply app/src/main/res/layout/activity_login.xml:26]: no. it's line 14

Comment (master-faz) [reply app/src/main/res/layout/activity_login.xml:26]: no, it's in line 14

**Review by mkaburis — APPROVED** [review]
Ready to merge!

**Pull request merged**